ablator/main/mp.py

```diff
@@ -56,6 +87,35 @@ def train_main_remote(
     fault_tollerant: bool = True,
     crash_exceptions_types: list[type] | None = None,
 ) -> tuple[ParallelConfig, dict[str, float] | None, TrialState]:
+    """
+    The trial job that will be executed remotely at a ray node. This is where model training happens.
+    In addition, experiment directory will be synchronized to the GCP and remote nodes.
```

Comment:
maybe mention the function where the sychronization happens is clearer for users to find out how it works